src/Sylius/Bundle/CoreBundle/EventListener/CustomerContextListener.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\EventListener;
```

Comment:
Why this is in `CoreBundle` and not in `UserBundle`? It doesn't really depend on anything from `Core` now.